.github/workflows/pr_check.yaml

```diff
@@ -7,12 +7,20 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        cypress-version: [^10, ^11, ^12]
+
     steps:
       - name: checkout
         uses: actions/checkout@v2
 
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
-      - run: npm install
+      - name: Install NPM dependencies for SUT
+        run: npm install
+      - name: Specify Cypress version based on testing strategy
```

Comment:
To be exact, here we don't really specify anything, the Cypress versions were specified during the definition of the strategy matrix. Here we should say something like `Install Cypress version {{ matrix.cypress-version }}`.

Reply:
To my surprise, it didn't work, now I see the following. :(

> Install Cypress version {{ matrix.cypress-version }}

Sorry, I didn't know. :(